### PR TITLE
Update WriteSoftware.md

### PR DESCRIPTION
**Pull request by pavlenex**
- Add Tip #2 to explain that any major change shoud first be communicated on GH via issue

- Change the last paragraph to explain that a review process is required and pull request merged to get a "contribution". Without it it's just a suggestion.

### CODE REVIEW

File: docs/Contribute/ContributeWrite/WriteSoftware.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 If you have trouble finding a string or contributing to the software stack, ask the [community](../../Community.md)
 :::
 
+
+::: tip
+If your text change is significant, contextual change [open an issue on GitHub](https://github.com/btcpayserver/btcpayserver/issues/new/choose) and explain what you'd like to change and why.:::
```
Comment (Zaxounette) [inline docs/Contribute/ContributeWrite/WriteSoftware.md:9]:
```suggestion
If your text change is significant or a contextual change, [open an issue on GitHub](https://github.com/btcpayserver/btcpayserver/issues/new/choose) and explain what you'd like to change and why.
:::
```

**Review by Zaxounette — CHANGES_REQUESTED** [review]

File: docs/Contribute/ContributeWrite/WriteSoftware.md (hunk 1 above)
Comment (pavlenex) [reply docs/Contribute/ContributeWrite/WriteSoftware.md:9]: That's not the context I want here. If I change is significant, contextual change is exactly what my goal is , what's not clear here?

Comment (Zaxounette) [reply docs/Contribute/ContributeWrite/WriteSoftware.md:9]: Oh right, you mean that it's significant **and** contextual.
I understood it as **or**, discard !

Comment (pavlenex) [reply docs/Contribute/ContributeWrite/WriteSoftware.md:9]: Yeah it's unclear, will just put significant

Comment (Zaxounette) [reply docs/Contribute/ContributeWrite/WriteSoftware.md]: Sorry, didn't get a popup for that comment.
ACK.

**Pull request merged**